mmv1/templates/terraform/examples/shared_vpc_internal_http_lb_with_mig_backend.tf.erb

```diff
@@ -0,0 +1,245 @@
+# project names
+variable "projects" {
```

Comment:
How is this supposed to work when running the integration test? The projects will not belong to the user who is running the tests

Reply:
Hi @slevenick I understand what you are suggesting. As you can understand, this is a multi-project code sample(Shared VPC LB - b/232285286). I see we have published a multi project sample at https://registry.terraform.io/providers/hashicorp/google/4.10.0/docs/resources/compute_shared_vpc_host_project 

Can you suggest any pointers for me?